allensdk/brain_observatory/behavior/write_nwb/extensions/stimulus_template/extension_builder.py

```diff
@@ -0,0 +1,52 @@
+import os.path
+
+from pynwb.spec import NWBNamespaceBuilder, export_spec, NWBGroupSpec, \
+    NWBDatasetSpec
+
+NAMESPACE = 'ndx-aibs-stimulus-template'
+
+
+def main():
+
+    ns_builder = NWBNamespaceBuilder(
+        doc="Stimulus images",
+        name=f"""{NAMESPACE}""",
+        version="""0.1.0""",
+        author="""Allen Institute for Brain Science""",
+        contact="""waynew@alleninstitute.org"""
+    )
+
+    ns_builder.include_type('ImageSeries', namespace='core')
+    ns_builder.include_type('TimeSeries', namespace='core')
+    ns_builder.include_type('NWBDataInterface', namespace='core')
+
+    stimulus_template_spec = NWBGroupSpec(
+        neurodata_type_def='StimulusTemplate',
+        neurodata_type_inc='ImageSeries',
```

Comment:
Hmm.... Instead of creating a whole new extension, why can't our stimulus templates be represented with two separate `ImageSeries` pynwb objects? 1 for the warped image and 1 for the unwarped?

Reply:
Tried it. requires 2 separate keys for the image_set_name as they can't both have the same name. It is much easier this way.

Reply:
Couldn't we just suffix `_warped` and `_unwarped` to the `image_set_name` to get unique names?

Reply:
Yes, but it causes all sorts of difficulty downstream. I tried this approach and backed out of it after realizing the difficulty it imposes.

Reply:
Okay, given that NWB team is likely to design and release a future core extension that will be a better fit for our image data and probably replace whatever we are using currently, I'm okay with this implementation.